File: java/src/main/java/com/saucelabs/simplesauce/SauceDataCenter.java
package com.saucelabs.simplesauce;

import lombok.Getter;

public enum SauceDataCenter {
    US_WEST("ondemand.us-west-1.saucelabs.com"),
    US_EAST("ondemand.us-east-1.saucelabs.com"),
    EU_CENTRAL("ondemand.eu-central-1.saucelabs.com");

    @Getter private final String value;

    SauceDataCenter(String value) {
        this.value = value;
    }
}

File: dotnet/Simple.Sauce/SauceDataCenter.cs
namespace Simple.Sauce
{
    public class SauceDataCenter
    {
        private SauceDataCenter(string value)
        {
            Value = value;
        }

        public string Value { get; set; }

        public static SauceDataCenter UsWest => new SauceDataCenter("ondemand.us-west-1.saucelabs.com/wd/hub");

        public static SauceDataCenter UsEast => new SauceDataCenter("ondemand.us-east-1.saucelabs.com/wd/hub");

        public static object EuCental => new SauceDataCenter("ondemand.eu-central-1.saucelabs.com/wd/hub");
    }
}